### DISCUSSION TITLE
Unifi CKg2+?

### DISCUSSION THREAD

**Discussion by cooper7692** [ep1cman/unifi-protect-backup#10] [General]
Does this work with the CKg2+

And does it suffer from the same issue as a similar tool that seems to overload the CK with to many requests and makes it reboot?

**Comment by ep1cman** [comment]
So long as it has Unifi Protect version 1.20 or higher it should work, however I don't own one (I use a UDM pro) so can't comment on compatibility/stability unfortunately. It would be good to hear back from you with how it works if you do try it.

**Comment by Sticklyman1936** [comment]
I'm using this with a CKg2+ with four attached cameras (G4 Doorbell, 2x G4 Bullet, G3 Instant). Everything is up-to-date (UniFi OS Version 2.3.15, Protect Version 1.21.0). It is working well, and I have yet to see a reboot.

I am also still running with https://github.com/danielfernau/unifi-protect-video-downloader which I have running as a cronjob every minute to pull and events that occur. This has been running for many months without issues too.

As I am running both, I am technically doubling the load on the CKg2+ so things seem pretty stable to me.
